application-services/custom/cloud-export-mqtt/res/configuration.toml

```diff
@@ -0,0 +1,91 @@
+[Writable]
```

Comment:
Default config can be removed. 
Need to add a README explaining how to run example using one of the profiles. i.e.
```
./app-service -p aws
of
./app-service -p azure
```

and maybe have something about the TransformToCloudFormat function is a simple example and real-world will need more use case specific??